### PR TITLE
Placeholder for apps and resources

### PR DESCRIPTION
**Pull request by v0ldemar01**

### CODE REVIEW

File: frontend/src/scenes/Apps/index.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import styles from './styles.module.scss';
-import { Form, FormControl } from 'react-bootstrap';
+import { Form, FormControl, InputGroup, Card } from 'react-bootstrap';
+import { Search } from 'react-bootstrap-icons';
```
Comment (uliana-lobanova) [inline frontend/src/scenes/Apps/index.tsx]: Use font-awesome icons instead of react-bootstrap-icons

**Review by uliana-lobanova — CHANGES_REQUESTED** [review]
Also: 

-  remove yarn.lock file, it shouldn't have been commited. 
- if you add some new packages, they should be added to package.json
- you haven't implemented placeholder for resources tab

File: frontend/src/scenes/Apps/index.tsx (hunk 1 above)
Comment (v0ldemar01) [reply frontend/src/scenes/Apps/index.tsx]: Fixing complete

**Review by uliana-lobanova — CHANGES_REQUESTED** [review]
yarn.lock file still needs to be removed

**Comment by v0ldemar01** [comment]
Done

**Review by uliana-lobanova — APPROVED** [review]

**Pull request merged**